account_partner_reconcile/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+Prohibits reconciliation for items:
+ - with different partners
+ - items with partner and without it
+This rule applies only for items with receivable and payable account type
+
+As on moment of instalation some items aleready was reconciled you can
+see them in menu Accounting/Adviser/Reconciled Journal Items above the rules
```

Comment:
`At installation, if some items were already reconciled, they can be seen in the menu Accounting/Adviser/Reconciled Journal Items above the rules.`
Better formulation ?